### PR TITLE
grpc quickstarts

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
**Check list**:

Your pull request:

- [X] targets the `development` branch
- [X] uses the `999-SNAPSHOT` version of Quarkus
- [X] has tests (`mvn clean test`)
- [X] works in native (`mvn clean package -Pnative`)
- [X] has native tests (`mvn clean verify -Pnative`)
- [ ] makes sure the documentation must not be updated
- [ ] links the documentation update pull request (if needed)
- [X] updates or creates the `README.md` file (with build and run instructions)
- [ ] For new quickstart, is located in the directory _component-quickstart_
- [X] For new quickstart, is added to the root `pom.xml` and `README.md`

What does `- [ ] For new quickstart, is located in the directory _component-quickstart_` mean? I have two quickstarts for grpc, should I rename their forlders somehow?

### CODE REVIEW

File: grpc-plain-text-quickstart/pom.xml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <quarkus-plugin.version>999-SNAPSHOT</quarkus-plugin.version>
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+        <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
+        <surefire-plugin.version>2.22.1</surefire-plugin.version>
+        <compiler-plugin.version>3.8.0</compiler-plugin.version>
+        <docker-plugin.version>0.28.0</docker-plugin.version>
```
Comment (cescoffier) [inline grpc-plain-text-quickstart/pom.xml:18]: is it used?

File: grpc-tls-quickstart/pom.xml [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <quarkus-plugin.version>999-SNAPSHOT</quarkus-plugin.version>
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+        <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
+        <surefire-plugin.version>2.22.1</surefire-plugin.version>
+        <compiler-plugin.version>3.8.0</compiler-plugin.version>
+        <docker-plugin.version>0.28.0</docker-plugin.version>
```
Comment (cescoffier) [inline grpc-tls-quickstart/pom.xml:18]: Is it used?

File: grpc-plain-text-quickstart/pom.xml (hunk 1 above)
Comment (michalszynkiewicz) [reply grpc-plain-text-quickstart/pom.xml]: no, fixed

File: grpc-tls-quickstart/pom.xml (hunk 2 above)
Comment (michalszynkiewicz) [reply grpc-tls-quickstart/pom.xml]: no, fixed

**Comment by cescoffier** [comment]
Waiting for CI.

**Comment by michalszynkiewicz** [comment]
CI run finished, merging

**Pull request merged**